examples/simple/src/posts/TagReferenceInput.tsx

```diff
@@ -57,14 +57,14 @@ const TagReferenceInput = ({
     source: string;
     label?: string;
 }) => {
-    const { change } = useForm();
+    const { setValue } = useFormContext();
```

Comment:
this should be documented in the upgrade guide

Reply:
Are we going to write an how to migrate from final-form to react-hook-form article ?